fe/fe-core/src/main/java/org/apache/doris/datasource/maxcompute/MaxComputeExternalCatalog.java

```diff
@@ -43,13 +48,16 @@
 import java.util.stream.Collectors;
 
 public class MaxComputeExternalCatalog extends ExternalCatalog {
+    private static final String endpointTemplate = "http://service.{}.maxcompute.aliyun-inc.com/api";
```

Comment:
Add aliyun doc link about this endpoint.

Reply:
ok . i will add it